CHANGES/7942.feature

```diff
@@ -0,0 +1,2 @@
+Improve sync performance with no-op when possible.
+To disable the no-op optimization use the ``optimize=False`` option on the ``sync`` call.
```

Comment:
I try to have the line wraps in the release notes occur at the bullet points. To do that, can these two lines be grouped like:

```
Improve sync performance with no-op when possible. To disable the no-op optimization use the
``optimize=False`` option on the ``sync`` call.
```